simpleLooping/listGreatestCommonDivisor.py

```diff
@@ -4,3 +4,33 @@
 # Ex:
 #   listGreatestCommonDivisor(-42, 402, -66, 354, 78, 426) # return 6
 #   listGreatestCommonDivisor(7883933, 91928167, 54151651) # return 1
+from typing import List
+
+def gcd(a: int, b: int) -> int:
+    while b:
+        a, b = b, a % b
+    return a
+
+def listGreatestCommonDivisor(integerList: List[int]) -> int:
+    lenIntegerList = len(integerList)
+    
+    for i in range(lenIntegerList):
+        if integerList[i] < 0:
+            integerList[i] *= -integerList[i]
+            
+    count = 0
+    for num in integerList:
```

Comment:
Now think about it, can you combine this loop and the loop above into one loop? So instead of

    for element in list:
        do something A

    for element in list:
        do something B
        
We can do:


    for element in list:
        do something A

        do something B

Reply:
But this is up to you to keep them separate or not.

Reply:
If you want decide to keep them separe, then it is better to keep this loop before the above loop. Because this is some kind of guarding condition where you check if there is at least 2 zeros to stop.

So it is meaning for the previous loop to do the "absolute" things if the next loop might terminate.

This will help reduce operational cost for computing as you terminate useless calculation.